src/main/scala/org/fortysevendeg/sparksftp/common/HiveUserData.scala

```diff
@@ -1,27 +1,32 @@
 package org.fortysevendeg.sparksftp.common
 
-import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.sql.{DataFrame, Dataset, Row, SparkSession}
 import SparkUtils._
+import cats.effect.IO
+import org.apache.spark.sql.types.IntegerType
 
 /**
  * Notes regarding operating with Hive:
  * Creating databases do not work in Dataproc: https://github.com/mozafari/verdictdb/issues/163
  * https://stackoverflow.com/questions/30664008/how-to-save-dataframe-directly-to-hive
  *
+ * Sample operations to perform on the user data
  */
+case class HiveUserData(
+    usersData: DataFrame,
+    salariesData: DataFrame,
+    userAndSalariesData: DataFrame
+)
+
 object HiveUserData {
 
-  /** Sample operations to perform on the user data
-   *
-   *
-   */
-  def persistUserData(sparkSession: SparkSession, users: DataFrame, salaries: DataFrame) = {
+  def persistUserData(sparkSession: SparkSession, users: DataFrame, salaries: DataFrame): Unit = {
 
-    persistDataFrame(sparkSession, users.select("ID", "name"), "user_data")
+    persistDataFrame(sparkSession, users.select("ID", "name", "age"), "user_data", Seq("age"))
```

Comment:
Could we use `List` instead of `Seq`?
```suggestion
    persistDataFrame(sparkSession, users.select("ID", "name", "age"), "user_data", List("age"))
```

Reply:
Changed in d504326